apps/daimo-mobile/src/view/screen/HomeScreen.tsx

```diff
@@ -357,4 +390,20 @@ const styles = StyleSheet.create({
     ...iconLabel,
     opacity: 0.5,
   },
+  checklistAction: {
+    flexDirection: "row",
+    justifyContent: "space-between",
+    alignItems: "center",
+    padding: 20,
+    borderRadius: 8,
+    borderWidth: 1,
+    borderColor: "#EEEEEE",
+    marginHorizontal: 24,
+    shadowColor: "#F3F3F3",
```

Comment:
talked to Michelle about shadows earlier and we settled on using [`ss.container.shadow`](https://github.com/daimo-eth/daimo/pull/905/files#diff-8c22dc3032962b6775c16c013325df38f25daafe470fb57e60e7938d2dfb4ef1R71) as in #905 -- want to match it?

Reply:
Sure. Do you recommend copying it over and using it here, or just waiting for the PR to be merged?